plothist/plotters.py

```diff
@@ -719,6 +639,13 @@ def plot_comparison(
             ax.set_ylabel(r"$\frac{" + h1_label + "}{" + h2_label + "}$")
 
         if ratio_uncertainty == "split":
+            np.seterr(divide="ignore", invalid="ignore")
+            h2_scaled_uncertainties = np.where(
```

Comment:
Why an uncertainty calculation here? This should be taken care of by `compute_comparison()`

Reply:
discussed in meeting